pac4j-core/src/main/java/org/pac4j/core/engine/DefaultCallbackLogic.java

```diff
@@ -81,7 +93,7 @@ public R perform(final C context, final Config config, final HttpActionAdapter<R
 
             final CommonProfile profile = client.getUserProfile(credentials, context);
             logger.debug("profile: {}", profile);
-            saveUserProfile(context, config, profile, multiProfile, renewSession);
+            saveUserProfile(context, config, profile, saveInSession, multiProfile, renewSession);
```

Comment:
Can we avoid passing the `saveInSession` property here to keep the current signature?

Reply:
Sure, should have done it.